test/shared/axiosHttpClientTests.ts

```diff
@@ -96,11 +96,8 @@ describe("axiosHttpClient", () => {
   });
 
   it("should allow canceling requests", async function() {
-    // ensure that a large upload is actually cancelled
-    this.timeout(2000);
-
     const controller = getAbortController();
-    const request = new WebResource(`${baseURL}/fileupload`, "POST", new Uint8Array(1024*1024*100), undefined, undefined, true, controller.signal);
+    const request = new WebResource(`${baseURL}/fileupload`, "POST", new Uint8Array(1024*1024*10), undefined, undefined, true, controller.signal);
```

Comment:
These sizes make mocha-chrome freak out and take 14 seconds to abort :/ Will want to do some stress testing separately.

Reply:
Do file an issue for that for tracking purpose.

Reply:
#109